### ISSUE TITLE
card_power_outlet not working

### ISSUE THREAD

**Issue by schumijo** [UI-Lovelace-Minimalist/UI#14]
Hi,

card_power_outlet is not working.

I think code should be :
```
card_power_outlet:
  template:
    - yellow
    - icon_info_bg
    - ulm_language_variables
  label: |-
    [[[ 
      if ((entity.state =='on') && (variables.ulm_card_power_outlet_consumption_sensor != '')) {
        return variables.ulm_on + ' • ' + states[variables.ulm_card_power_outlet_consumption_sensor].state + 'W';
      } else {
        return variables.ulm_off;
      }
    ]]]
```

**Comment by saxel** [comment]
The documentation shows ulm_card_power_outlet_consumption_sensor as required = false.

If I don't have power consumption switches then I don't want to specify its variable in my views:
```
- type: 'custom:button-card'
  template: card_power_outlet
  variables:
    ulm_card_power_outlet_consumption_sensor: ''
  entity: switch.power_outlet_livingroom
  name: Power Outlet Livingroom
```

Instead I could write:
```
- type: 'custom:button-card'
  template: card_power_outlet
  entity: switch.power_outlet_livingroom
  name: Power Outlet Livingroom
```

That would require template in button_card_template.yaml to be something like
```
card_power_outlet:
  template:
    - icon_info_bg
    - yellow
    - ulm_language_variables
  variables:
    ulm_card_power_outlet_consumption_sensor:
  label: |-
    [[[
      if (entity.state === 'on' && variables.ulm_card_power_outlet_consumption_sensor !== null) {
        return variables.ulm_on + ' • ' + states[variables.ulm_card_power_outlet_consumption_sensor].state + 'W';
      } else if (entity.state === 'on') {
        return variables.ulm_on;
      } else {
        return variables.ulm_off;
      }
    ]]]
```

**Comment by Paddy0174** [comment]
Closed with commit bd6cb848dbbbf37911f721f140daa8b9dd6013ea
